www/docs/client/links/httpBatchStreamLink.md

```diff
@@ -15,6 +15,10 @@ We have prefixed this as `unstable_` as it's a new API, but you're safe to use i
 
 > All usage and options are identical to [`httpBatchLink`](./httpBatchLink.md).
 
+:::note
+If you require the ability to change/set response headers (which includes cookies) from within your procedures, make sure to use `httpBatchLink` instead! [Read more](https://trpc.io/docs/client/links/httpBatchLink).
+:::
```

Comment:
Should state the reason you have to do this is cause streaming doesn't support setting headers after the response has started streaming

Reply:
> Should state the reason you have to do this is cause streaming doesn't support setting headers after the response has started streaming

Thanks for pointing that out Julius, I've added a snippet to the end :)